### PR TITLE
feat: add model selection for LLM summarization in app UI

### LINKED ISSUE #56: add more models for LLM summary fetching

### PR DESCRIPTION
**Pull request by ANGorbachev**
Fixes #56

### CODE REVIEW

File: src/run_app.py [hunk 1]
```diff
@@ -77,6 +78,19 @@
         summary_checkbox = st.checkbox("🔎 Аннотирование текста", value=False)
         transcribe_text = ""
 
+        if summary_checkbox:
+            model_options: Dict[str, str] = {
```
Comment (khit-mle) [inline src/run_app.py:82]: We should also add our previous default option: `"openai/gpt-4-turbo-preview"`

**Review by khit-mle — CHANGES_REQUESTED** [review]
Please, consider adding `"openai/gpt-4-turbo-preview"` as per my comment.

**Review by DanilKhardi — CHANGES_REQUESTED** [review]
I agree with @khit-mle, with these changes we've lost ability to provide our user with `"openai/gpt-4-turbo-preview"` model's summarization output if the user prefers that model.

**Review by khit-mle — APPROVED** [review]
## Code Review of [PR-57](https://github.com/mlteamurfu2325/practicum-s1/pull/57) by @khit-mle 

### Overview
- [x] The code follows the project's coding conventions and style guide.
- [x] The code is well-structured, readable, and maintainable.
- [x] The code is properly documented with clear comments and docstrings.
- [x] The code is efficient and optimized for performance, if applicable.
- [x] The code handles edge cases and potential errors gracefully.
- [x] The code is free of any security vulnerabilities or potential risks.

### Functionality
- [x] The code implements the required functionality as described in the PR description.
- [x] The code handles all expected inputs and outputs correctly.
- [x] The code integrates well with the existing codebase and does not introduce any conflicts.
- [x] The code does not introduce any new bugs or regressions.

### Testing
- [x] The code passes all existing tests and any newly added tests.
- [x] The tests cover a sufficient range of scenarios and edge cases (needed if new tests are added).
- [x] The tests are well-structured, readable, and maintainable (needed if new tests are added).

### Documentation
- [x] The code includes clear and concise comments explaining complex logic or algorithms.
- [x] The code includes docstrings for functions, classes, and modules, following the project's documentation standards.
- [x] The PR description provides a clear explanation of the changes made and their purpose.
- [x] Any necessary updates to the project's documentation (README, API docs, etc.) are included.

### Suggestions and Improvements
- [ ] Are there any areas where the code could be further optimized or improved?
- [ ] Are there any best practices or design patterns that could be applied to enhance the code?
- [ ] Are there any potential edge cases or error scenarios that have not been considered?
- [ ] Are there any additional tests that could be added to improve code coverage?

### Other Comments
@ANGorbachev Thank you for your PR#57.

## Overall Recommendation:
- [x] Approve
- [ ] Request Changes
- [ ] Comment Only

**Review by DanilKhardi — APPROVED** [review]
## Code Review of [PR-57](https://github.com/mlteamurfu2325/practicum-s1/pull/57) by @DanilKhardi

### Overview
- [x] The code follows the project's coding conventions and style guide.
- [x] The code is well-structured, readable, and maintainable.
- [x] The code is properly documented with clear comments and docstrings.
- [x] The code is efficient and optimized for performance, if applicable.
- [x] The code handles edge cases and potential errors gracefully.
- [x] The code is free of any security vulnerabilities or potential risks.

### Functionality
- [x] The code implements the required functionality as described in the PR description.
- [x] The code handles all expected inputs and outputs correctly.
- [x] The code integrates well with the existing codebase and does not introduce any conflicts.
- [x] The code does not introduce any new bugs or regressions.

### Testing
- [x] The code passes all existing tests and any newly added tests.
- [x] The tests cover a sufficient range of scenarios and edge cases (needed if new tests are added).
- [x] The tests are well-structured, readable, and maintainable (needed if new tests are added).

### Documentation
- [x] The code includes clear and concise comments explaining complex logic or algorithms.
- [x] The code includes docstrings for functions, classes, and modules, following the project's documentation standards.
- [x] The PR description provides a clear explanation of the changes made and their purpose.
- [x] Any necessary updates to the project's documentation (README, API docs, etc.) are included.

### Suggestions and Improvements
- [ ] Are there any areas where the code could be further optimized or improved?
- [ ] Are there any best practices or design patterns that could be applied to enhance the code?
- [ ] Are there any potential edge cases or error scenarios that have not been considered?
- [ ] Are there any additional tests that could be added to improve code coverage?

### Other Comments
@ANGorbachev Everything seems to be in order now, thanks!

## Overall Recommendation:
- [x] Approve
- [ ] Request Changes
- [ ] Comment Only
